components/brave_ads/browser/ads_service_impl.cc

```diff
@@ -55,6 +54,7 @@
 #include "brave/components/brave_news/common/pref_names.h"
 #include "brave/components/brave_rewards/browser/rewards_service.h"
 #include "brave/components/brave_rewards/common/mojom/rewards.mojom-forward.h"
+#include "brave/components/deprecated_channel_name/channel_name.h"
```

Comment:
I cant see where this DEP was added. cc @DJAndries

Reply:
https://github.com/brave/brave-core/blob/b87d9880cb65af421b13b20c66fc9212d90c6a4c/components/brave_ads/browser/BUILD.gn#L45

Reply:
Damn... I am blind :-) Thanks